agent/src/issuer.ts

```diff
@@ -52,18 +53,17 @@ export const credentialRequestToCredentialMapper: OpenId4VciCredentialRequestToC
   async ({
     credentialsSupported,
     credentialRequest,
+    issuanceSession,
     // FIXME: it would be useful if holderBinding would include some metadata on the key type / alg used
     // for the key binding
     holderBinding,
   }) => {
     const credentialSupported = credentialsSupported[0];
+    const issuanceSessionRepository = agent.dependencyManager.resolve(
+      OpenId4VcIssuanceSessionRepository
+    );
 
-    // FIXME: issuanceSession will be passed to this method from 0.5.1 onward
-    //  https://github.com/openwallet-foundation/credo-ts/pull/1795
-    const issuanceSession =
-      await agent.modules.openId4VcIssuer.findIssuanceSessionForCredentialRequest(
-        { credentialRequest }
-      );
+    // not sure if this check is needed anymore
     if (!issuanceSession) throw new Error("Issuance session not found");
```

Comment:
issuanceSession will always be defined, we do need the second check (for issuanceMetadata) though